src/main/groovy/com/netflix/nebula/lint/plugin/LintGradleTask.groovy

```diff
@@ -35,15 +44,16 @@ class LintGradleTask extends DefaultTask {
 
     @TaskAction
     void lint() {
-        def violations = new LintService().lint(project, onlyCriticalRules).violations
+        def violations = new LintService().lint(project, getOnlyCriticalRules()).violations
                 .unique { v1, v2 -> v1.is(v2) ? 0 : 1 }
 
-        (listeners + new GradleLintPatchAction(project) + new GradleLintInfoBrokerAction(project) + consoleOutputAction).each {
+        (getListeners() + new GradleLintPatchAction(project) + new GradleLintInfoBrokerAction(project) + consoleOutputAction).each {
             it.lintFinished(violations)
         }
 
     }
 
+    @Internal
```

Comment:
Reading through https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/Internal.html, is this marked as such so we only use this 1x? Or another reason?

Reply:
imo, this portion is not a task input or output file. So it should be marked as `internal` so  is not to be taken into account for up-to-date checking.

thoughts?

Reply:
Yeah, based on https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/package-summary.html -> Annotation Types Summary, that looks like the best option. It would be nice if it didn't need an annotation. I suppose that it means more "task lifecycle shouldn't worry about this field/property" instead of "internal usage only"